ironfish/src/mining/stratum/messages.ts

```diff
@@ -20,6 +20,7 @@ export type MiningDisconnectMessage =
 
 export type MiningSubscribeMessage = {
   version?: number
+  name: string
```

Comment:
This should not be required or will require a stratum bump. Miner names should be an opt in feature.

```suggestion
  name?: string
```